backend/src/main/java/de/neuefische/app/playlist/PlaylistService.java

```diff
@@ -0,0 +1,21 @@
+package de.neuefische.app.playlist;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class PlaylistService {
+
+    private final PlaylistRepository playlistRepository;
+
+    public Optional<PlaylistDTO> savePlaylist(PlaylistData playlistData){
```

Comment:
In my opinion the Service should not know about the DTO. DTOs are only relevant in the controller

Reply:
Good to know! I will change that in the following commits, when I have a PlaylistController